.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
+dist: xenial
 python:
-- 3.6
+  - "3.6"
+  - "3.7"
 install: pip install -U tox-travis
 script: tox
 deploy:
```

Comment:
with this change, a deploy will be triggered on both python 3.6 and 3.7, so the second one will fail. We chould use a matrix, something like that:

https://github.com/PacktPublishing/Migrating-from-Python-2-to-Python-3/blob/master/chapter08/packt-calc/.travis.yml

So we could add tests on macos and windows at the same time.